rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search/120_batch_reduce_size.yml

```diff
@@ -1,4 +1,7 @@
 setup:
+  - skip:
```

Comment:
Outside of this one I think the remaining TODOs are not blockers to reviewing and/or merging this. They're mostly concerned with test follow-ups and possible follow-on improvements.

Reply:
Right this one was a little tricky. We use the test in all kinds of configurations and it's not deterministic what the node configuration looks like. Depending on how the coordinating node + data node + shard assignment layout randomly works out there either is a num_reduce_phases field or not.

Weirdly enough, I wasn't able to find a way of making the matcher optional? Unless you know one, my suggestion would be to leave this ignored and deal with it in a follow-up. The test has little to no added value otherwise since it's just some random aggregation run that we test elsewhere as well?

Reply:
sounds good, seems like this tests batch_reduce_size, we should remove it as a follow-up perhaps or just ensure that the param is accepted but no-op in batched execution.